src/json/serialization.cr

```diff
@@ -126,6 +126,27 @@ module JSON
   # field, and the rest of the fields, and their meaning, depend on its value.
   #
   # You can use `JSON::Serializable.use_json_discriminator` for this use case.
+  #
+  # ### `after_initialize` method
+  #
+  # after_initialize is a method that runs after initialization and can be used
```

Comment:
I'd suggest to use the typical format for referencing instance methods, hash prefix and wrapped in backticks.
```suggestion
  # `#after_initialize` is a method that runs after initialization and can be used
```